ms-common-auth/src/main/java/net/trajano/ms/auth/spi/ClientValidator.java

```diff
@@ -3,7 +3,10 @@
 import java.net.URI;
 import java.text.ParseException;
 
+import net.trajano.ms.auth.token.GrantTypes;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.trajano.ms.auth.token.GrantTypes'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)